docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -19,331 +19,362 @@ tags:
-Each key is unique to a single identity and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+This document introduces Temporal Cloud's API key features:
 
-The process is as follows:
+- [API key overview](#overview)
+- [Global Administrator API key management](#manage-api-keys)
+- [User API key management](#user-api-keys)
+- [Manage API keys for Service Accounts](#serviceaccount-api-keys)
+- [API keys for Namespace authentication](#namespace-authentication)
+- [Use API keys to authenticate](#using-apikeys)
+- [API key best practices](#best-practices)
+- [Troubleshoot your API key use](#troubleshooting)
+- [API keys: Frequently Asked Questions](#faqs)
 
-```text
-API Key (authentication) → Identity (User or Service Account) → RBAC (authorization)
-```
+## API key overview {#overview}
 
-API Keys support 2 main use cases:
+Temporal Cloud API keys offer industry-standard identity-based authentication for Temporal users and [Service Accounts](/cloud/service-accounts).
+Each key is a unique identity.
+It is linked to role-based access control (RBAC) settings to ensure secure and appropriate access.
 
-- _**Cloud Operations Automation**_: API Keys can be used as the authentication mechanism for Temporal Cloud operational tools ([Temporal Cloud CLI (tcld)](/cloud/tcld), [the Cloud Ops API](/ops), [Terraform](/production-deployment/cloud/terraform-provider)) to manage your Temporal Cloud account, Namespaces, certificates, and user identities.
-  - API Keys for use in Cloud Operations Tools are in Public Preview
-- _**Namespace Authentication**_: API Keys can be used as the Namespace authentication mechanism for executing and managing workflows from the SDK and Temporal CLI as an alternative to using mTLS-based authentication.
-  - API Keys for use in Namespace Authentication are in Pre Release
+The authentication process follows this pathway:
 
-API Keys are an opt-in feature that must be enabled by a Global Administrator for use. For instructions on API Key Management for your organization, see [Global Administrator API Key Management](#ga-manage-api-keys).
+```text
+API key (authentication) → Identity (user or Service Account) → RBAC (authorization)
+```
 
-API Keys are supported for Users and Service Accounts.
+### API key use cases
 
-- Users may create, delete, and update API key access using the Cloud Web UI or the `tcld` CLI tool.
-  Global Administrators have permission to manage user API keys using the same interfaces.
-- Global Administrators can create, delete, and update access to API Keys for Service Accounts.
+API keys are well suited for the following scenarios:
 
-API Keys can be used as an authentication mechanism with:
+- _**Cloud operations automation**_:
+  API keys work with most Temporal Cloud operational tools, including the [`tcld`](/cloud/tcld), [Cloud Ops APIs](/ops), and [the Terraform provider](/production-deployment/cloud/terraform-provider).
```

Comment:
Not sure it's meant to be "the" `tcld` ?

Reply:
the `tcld` CLI utility, but I think `tcld` on its own is just fine. Fixing.

Reply:
Removed "the".